recipe/meta.yaml

```diff
@@ -16,11 +16,12 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.7
     - setuptools
     - pip
   run:
-    - python >=3.6
+    # this avoids the need to include importlib-metadata
+    - python >=3.7
```

Comment:
Since conda-forge has stopped building new packages with even python 3.7 a while ago, I think it's quite safe to stop supporting python 3.6 for easier package management even if upstream still does.